scipy/spatial/tests/test_distance.py

```diff
@@ -1245,15 +1256,22 @@ def test_squareform_one_binary_vector(self):
         rv = squareform(v)
         assert_equal(rv.shape, (2,2))
         assert_(rv[0,1])
+        assert_equal(rv.dtype.kind, 'b')
 
     def test_squareform_2by2_matrix(self):
         A = np.zeros((2,2))
         A[0,1] = 0.8
         A[1,0] = 0.8
-        rA = squareform(np.array(A, dtype='double'))
+        rA = squareform(np.array(A, dtype=np.float))
```

Comment:
``np.float`` is better not used, it's an outdated alias:

    >>> float is np.float
    True